src/tests/accessIndex/accessIndex/accessindex.test.js

```diff
@@ -4,7 +4,9 @@ const patient1Resource = require('./fixtures/Patient/patient1.json');
 
 // expected
 const expectedAuditEventResources = require('./fixtures/expected/expected_AuditEvent.json');
+const expectedAuditEventResourcesThedcare = require('./fixtures/expected/expected_AuditEvent_thedcare.json');
 const expectedAuditEventWithoutAccessIndexResources = require('./fixtures/expected/expected_AuditEvent_without_access_index.json');
+const expectedAuditEventWithoutAccessIndexResourcesThedcare = require('./fixtures/expected/expected_AuditEvent_thedcare.json');
 const expectedAuditEventResourcesAccessIndex = require('./fixtures/expected/expected_AuditEvent_access_index.json');
```

Comment:
Can we remove unused imports and also delete their files?

Reply:
Removed unused import on line 9. Line 7 is used in the test code.

Reply:
Removed unused import.